packages/rocketchat-lib/server/methods/leaveRoom.js

```diff
@@ -27,7 +27,15 @@ Meteor.methods({
 				throw new Meteor.Error('error-you-are-last-owner', 'You are the last owner. Please set new owner before leaving the room.', { method: 'leaveRoom' });
 			}
 		}
+		if (room.automatic) {
```

Comment:
Sir, not able to use callback because I am using  ``` RocketChat.removeUserFromRoom``` 
in my another function ``` remove_user_from_automatic_channel ``` and there I do not want that this callback to fire